code/modules/mob/mob.dm

```diff
@@ -1252,13 +1252,21 @@ Use this proc preferably at the end of an equipment loadout
 	A.examine(src)
 
 
-/mob/living/verb/verb_pickup(obj/I in view(1))
+/mob/living/verb/verb_pickup(obj/I in objects_in_view(usr, 1))
```

Comment:
So can you not just replace `view()` with `oview()`?

Reply:
Wanted to add some sanity, as it was giving you a list of every object in view/oview, including lockers, hidden pipes, etc.

Reply:
Fair enough
It might make sense to use `oview()` in the proc you wrote but maybe there are some circumstances in which you should be able to pick up something directly inside you
Up to you

Reply:
it loops through all turfs in view, then gets their contents, so you'd be able to pick up something off the floor that you're standing on, but not something within yourself